pkgs/os-specific/linux/fswebcam/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl, libv4l, gd }:
+
+stdenv.mkDerivation rec {
+  name = "fswebcam-20140113";
+
+  src = fetchurl {
+    url = "http://www.sanslogic.co.uk/fswebcam/files/${name}.tar.gz";
+    sha256 = "3ee389f72a7737700d22e0c954720b1e3bbadc8a0daad6426c25489ba9dc3199";
+  };
+
+  buildInputs =
+    [ libv4l gd ];
+
+  meta = {
+    description = "fswebcam is a neat and simple webcam app";
```

Comment:
Don't use the name of app in `meta.description`. It is meant to be a terse description.

You can use `LongDescription` to write a more detailed description.